index.html

```diff
@@ -307,6 +307,55 @@ <h2>About me</h2>
       </div>
     </main>
 
+    <footer class="about flex-column">
+      <h2 id="about__contactUs">CONTACT US</h2>
+      <p id="about__needHelp">
+        I'm always interested in hearing about new projects, so if you'd like to chat please get in.
+      </p>
+      <form
+        action="https://formspree.io/f/mbjqvypj"
+        method="post"
+        class="flex-column"
+      >
+        <div class="input">
+          <input
+            type="text"
+            name="name"
+            placeholder="Full name"
+            maxlength="30"
+            class="name"
+            required
+          />
+        </div>
+        <div class="input">
+          <input
+            type="email"
+            name="email"
+            placeholder="jeremy@gmail.com"
```

Comment:
- [ ] Please update the placeholder to match the Figma design as mentioned by the previous reviewer. :point_up: